src/main/java/org/sagebionetworks/web/client/widget/entity/WikiPageWidgetViewImpl.java

```diff
@@ -462,6 +462,8 @@ public void invoke() {
 				});
 				
 				add(markdownEditorWidget.asWidget());
+				//also add the wiki attachments dialog
+				add(wikiAttachments.asWidget());

```

Comment:
should be manager, where you can add attachments, delete, and choose to save or discard changes.